frontend/src/icons/alert_icon_small.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this actually used anywhere? I could not find any reference to this

Reply:
Oh right it was used when customers needed to have representatives. Not needed anymore 👍